packages/core/src/telemetry/service.ts

```diff
@@ -146,13 +199,18 @@ export class TelemetryService {
     const serializedEvents = JSON.stringify(eventsWithContext);
 
     const telemetryEventsFilePath = path.join(
-      this.options.ponderDir,
+      os.tmpdir(),
       "telemetry-events.json",
     );
+    const detachedFlushScriptPath = path.join(os.tmpdir(), "detached-flush.js");
+
+    //Write remaining events and flush script to tmp files
     fs.writeFileSync(telemetryEventsFilePath, serializedEvents);
+    fs.writeFileSync(detachedFlushScriptPath, this.detachedFlushScript());
 
+    //Spawn child
     child_process.spawn(process.execPath, [
-      path.join(__dirname, "detached-flush.js"),
+      detachedFlushScriptPath,
       this.options.telemetryUrl,
       telemetryEventsFilePath,
```

Comment:
Not blocking, but I am interesting in a solution where we just pass the `serializedEvents` string as a CLI arg to the child process rather than write + read from a file.